src/preproc.rs

```diff
@@ -109,103 +109,6 @@ fn trim_line(input: &str) -> String {
 
 // Aggressive preprocessors
 
-fn lcs_substr(fstr: &str, sstr: &str) -> Option<String> {
```

Comment:
This is short-term dead code to band-aid this: https://github.com/amzn/askalono/issues/42
Mind adding that back for now? I'm fine with directives to silence the warning; I'll ensure that a crates.io version won't have any annoying warnings.

Reply:
Done!